cross-wiki-weekend-2020.php

```diff
@@ -0,0 +1,475 @@
+			</tr>
+			<tr>	
+				<td style="padding: 8px 4px; text-align: center; width: 25%;"><a style="color:#53C651;" href="https://zelda.gamepedia.com/Zelda_Wiki:Cross-Wiki_Weekend"><img src="http://www.niwanetwork.org/images/members/zeldawiki.png" alt="Zelda Wiki" width="90px"><br/><b>Cross-Wiki Guide</b></a></td>
+				<td style="padding: 8px 4px; text-align: center; width: 25%;"> </td>
+				<td style="padding: 8px 4px; text-align: center; width: 25%;"> </td>
+				<td style="padding: 8px 4px; text-align: center; width: 25%;"> </td>
+			</tr>
+		</table>
+
+		<h3 style="text-align: center;">Italian Wikis</h3>
+		<table style="margin: 5px; width: 100%;">
+			<tr>
+				<td style="padding: 8px 4px; text-align: center; width: 25%;"><a style="color:#53C651;" href="https://wiki.pokemoncentral.it/Meta:Cross-Wiki_Weekend"><img src="http://niwanetwork.org/images/members/pokemoncentral.png" alt="Pokémon Central Wiki" width="90px"><br/>Cross-Wiki Guide</a></td>
+				<td style="padding: 8px 4px; text-align: center; width: 25%;"><a style="color:#53C651;" href="https://www.mariocastle.it/wiki/Super_Mario_Wiki:Cross-Wiki_Weekend"><img src="http://niwanetwork.org/images/members/mariowikiit.png" alt="Super Mario Wiki (IT)" width="90px"><br/>Cross-Wiki Guide</a></td>
+				<td style="padding: 8px 4px; text-align: center; width: 25%;"><a style="color:#53C651;" href="https://www.xenopedia.it/Meta:Cross-Wiki_Weekend"><img src="http://niwanetwork.org/images/members/xenopediait.png" alt="Xenopedia (IT)" width="90px"><br/>Cross-Wiki Guide</a></td>
+			</tr>
+		</table>
+
+		<br>
+		<div style="text-align: center; width: 100%;"><i>WarsWiki, and Pikmin Fanon are not participating this year.</i></div>
+		<br>
+		
+		<h1 id="Form">Form</h1>
+		<style>
+			.button {
+				-webkit-appearance: button;
+				-moz-appearance: button;
+				appearance: button;
+				background-color: #7F7F7F;
+				border: none;
+				padding: 15px 15px;
+				text-align: center;
+				text-decoration: none;
+				display: inline-block;
+				font-size: 16px;
+			}
+		</style>
+
+		At the end of the event, please report your contributions by filling out the following form by <b>September 30<sup>th</sup></b>.
+		Filling out the form is also how you enter yourself for a chance to win the prize.
+		<br><br>
+		<div style="width: 100%; text-align: center;">
+			<div class="button">Submit Contributions</div>
+		</div>
+
+		<br><br>
+
+		<h1 id="Drawing">Drawing</h1>
+		The winner of the contest is determined by random draw.
+		Each person's contributions are rated on a scale of 1 to 5; the more substantial one's contributions, the greater the chance of winning.
+		Be sure to make some substantive edits for a better shot at the prize!
+		
+		<br><br>
+
+		While event organizers are free to participate, they are disallowed from entering the drawing and winning the prize.
+
+		<br><br>
+		<!--
```

Comment:
Let's remove this Conclusion section (even if commented out), we can re-create it at the end of the event